src/main/java/com/couchbase/lite/replicator/Replication.java

```diff
@@ -364,7 +400,8 @@ public ReplicationStatus getStatus() {
             return ReplicationStatus.REPLICATION_OFFLINE;
         } else if (replicationInternal.stateMachine.isInState(ReplicationState.IDLE)) {
             return ReplicationStatus.REPLICATION_IDLE;
-        } else if (replicationInternal.stateMachine.isInState(ReplicationState.STOPPED)) {
+        } else if (replicationInternal.stateMachine.isInState(ReplicationState.INITIAL) ||
+                replicationInternal.stateMachine.isInState(ReplicationState.STOPPED)) {
```

Comment:
This issue was found during writing replicator state test line. Please see following link.
https://github.com/couchbase/couchbase-lite-android/blob/feature/issue_894_docIDs/src/androidTest/java/com/couchbase/lite/replicator/ReplicationTest.java#L124